pkg/firewall/client.go

```diff
@@ -119,3 +123,7 @@ func bastionFirewallPolicyRuleName(clusterName string) string {
 func isAlreadyExistError(err error) bool {
 	return strings.Contains(err.Error(), "already exists")
 }
+
+func isNotFoundError(err error) bool {
+	return strings.Contains(err.Error(), "was not found")
```

Comment:
I'm afraid this would be too brittle. Maybe we can check if 404 is part of the message?

Reply:
what ever you say boss